osu.Game/Screens/Menu/OsuLogo.cs

```diff
@@ -229,7 +229,7 @@ public OsuLogo()
         }
 
         /// <summary>
-        /// Schedule a new extenral animation. Handled queueing and finishing previous animations in a sane way.
+        /// Schedule a new external animation. Handled queuing and finishing previous animations in a sane way.
```

Comment:
Same here for `queuing`, https://github.com/ppy/osu/pull/5883#discussion_r319149348